project/XNATDataImport.py

```diff
@@ -0,0 +1,64 @@
+import os
+import logging
+import pandas as pd
+import numpy as np
+
+from utils.tools import DataBuilderXNAT
```

Comment:
Could XNATDataImport be merged/refactored with DataBuilderXNAT? What's the benefit of having them separate?

A merged version is easier to get rid of/ignore for not image-based projects.

Reply:
Different functionalities so keep separate.

Reply:
make plans to move both to CSC-XNAT package